src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -4620,8 +4616,9 @@ private static async Task<Process> StartProcess(string hostname, string benchmar
                 var arguments = process.StartInfo.Arguments;
 
                 // The executable should be in the same folder as the agent since it references the Console project
-                process.StartInfo.FileName = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Microsoft.Crank.JobObjectWrapper.exe");
-                process.StartInfo.Arguments = filename + " " + arguments;
+                // Use 'dotnet exe .dll' to use the current default dotnet version or the tests could fail if the .exe doesn't match what version is available locally
```

Comment:
```suggestion
                // Use 'dotnet exec .dll' to use the current default dotnet version or the tests could fail if the .exe doesn't match what version is available locally
```
Noticed a small type here. I didn't realise that `dotnet exec .dll` was a thing until now, It seems that it is equivalent to just doing `dotnet .dll` but I guess it doesn't hurt to use `exec`